src/worker-type.js

```diff
@@ -824,14 +824,28 @@ WorkerType.prototype.determineSpotBids = function(managedRegions, pricing, chang
         .filter(r => _.includes(managedRegions, r.region))
         .map(r => r.region);
 
+    let configuredAzs = {};
+    this.availabilityZones.forEach(az =>
+      az.region in configuredAzs ?
```

Comment:
An if statement would read a little more clearly here, would you mind switching to that?

Reply:
Hah, hassan and I were trying to find a one-liner for it instead of an if.  but I can change it back.